code_challenges/code_wars/reverse_string.py

```diff
@@ -0,0 +1,15 @@
+# https://www.codewars.com/kata/5264d2b162488dc400000001
+
+
+def reverse_larger_strings(sentence):
+    sentence = sentence.split()
+    reversed_words = ""
+    for word in sentence:
+        if len(word) < 5:
+            reversed_words += f"{word} "
+        else:
+            reversed_word = ""
+            for letter in word:
+                reversed_word = letter + reversed_word
+            reversed_words += reversed_word + " "
+    return reversed_words.strip()
```

Comment:
`strip` is fine here since you know the string doesn't have leading white space.

Remember that `rstrip` and `lstrip` are also available when you might not want to remove both leading and trailing. 

[str.strip](https://docs.python.org/3/library/stdtypes.html?highlight=strip#str.strip)
[str.rstrip](https://docs.python.org/3/library/stdtypes.html?highlight=strip#str.rstrip)
[str.lstrip](https://docs.python.org/3/library/stdtypes.html?highlight=strip#str.lstrip)